katas/content/teleportation/standard_teleportation_protocol/index.md

```diff
@@ -0,0 +1,8 @@
+Put together the steps implemented in previous three exercises to implement the full teleportation protocol.
+
+**Input:** 
+1. The two qubits qAlice and qBob in $\ket{0}$ state.
+2. The message qubit qMessage in the state $\ket{\psi}$ to be teleported.
+
+**Output:**  
```

Comment:
For operations with unit return, we use goal rather than output, since they act via side effects
```suggestion
**Goal:**
```